server/docker-compose.yaml

```diff
@@ -29,10 +29,16 @@ services:
         image: yagna-api
         environment:
             - PYTHONUNBUFFERED=1
-            - YAGNA_URL=http://yagna:6000/
+            - YAGNA_API_URL=http://yagna:6000
+            - GSB_URL=unix:/app/yagna/yagna.sock
+            - YAPAPI_SUBNET_TAG=galatea
+            - YAPAPI_DRIVER=zksync
+            - YAPAPI_NETWORK=rinkeby
+            - PATH=/bin:/usr/bin:/usr/local/bin
```

Comment:
@pnowosie ?

Reply:
I can admit I don't understand this. But the similar we do for the `python` ran inside a vm (process run by shell vs directly). 
Reason for setting a path was `Yapapi` could not see the `gftp` but once I log into the container everything was available and PATH contained even more locations... So by setting the path Yapapi can now use gftp, and some other locations for python...

Reply:
I'm not sure if I understand the answer : ) My point is that:
* if this is not necessary, it should be removed
* if this is neither a thing that could be changed by docker-compose user (like YAPAPI_SUBNET_TAG), nor something related to between-container communication, it belongs to Dockerfile not docker-compose

Reply:
> if this is not necessary, it should be removed

It is necessary, otherwise yapapi won't be able to find `gftp` binary.

> if this is neither a thing that could be changed by docker-compose user (like YAPAPI_SUBNET_TAG), nor something related to between-container communication, it belongs to Dockerfile not docker-compose

Agreed. It belongs in the Dockerfile.